src/openrct2-android/app/build.gradle

```diff
@@ -77,41 +81,19 @@ android {
     }
 }
 
-apply plugin: 'de.undercouch.download'
+delete {
+    delete 'src/main/assets/openrct2'
+}
 
-android.applicationVariants.all { variant ->
-    variant.mergeAssets.doLast {
-        copy {
-            from '../../../data'
-            into "$variant.mergeAssets.outputDir/data"
-        }
-        download {
-            src 'https://github.com/OpenRCT2/openrct2-dependencies-android/releases/download/v0.8/g2.dat'
-            dest "$variant.mergeAssets.outputDir/data"
-        }
-        download {
-            src 'https://github.com/OpenRCT2/title-sequences/releases/download/v0.1.2c/title-sequences.zip'
-            dest new File(buildDir, 'title-sequence.zip')
-        }
-        copy {
-            from zipTree(new File(buildDir, 'title-sequence.zip'))
-            into "$variant.mergeAssets.outputDir/data/title"
-        }
-        download {
-            src 'https://github.com/OpenRCT2/objects/releases/download/v1.0.12/objects.zip'
-            dest new File(buildDir, 'objects.zip')
-        }
-        copy {
-            from zipTree(new File(buildDir, 'objects.zip'))
-            into "$variant.mergeAssets.outputDir/data/object"
-        }
-    }
```

Comment:
These are essential steps for generating (release) builds) in CI jobs. Why were these steps removed? It seems the code that replaces it does not do the same thing.

Reply:
The download plugin does not work anymore. My best guess is that it doesn't support TLSv1.2 which GitHub now requires. These assets are all duplicated elsewhere in the build, so I just took them from there.

variant.mergeassets.outputDir no longer exists and I'm not sure why it was used in the first place - these files are copied out of the APK and into /sdcard/openrct2 and that's it, so I just placed the files into the standard assets folder which is included in all builds.